inductiva/_cli/main.py

```diff
@@ -0,0 +1,54 @@
+"""Main CLI entrypoint."""
```

Comment:
You can also add an example commands you have in the PR description here.

Reply:
That's true, but then I think as we move forward we will still need to add a small documentation in the readme and keep it up to date.